src/tests/calculator/operate.test.js

```diff
@@ -0,0 +1,25 @@
+import operate from '../../logic/operate';
+
+describe('Test each mathematical operation in operate.js', () => {
+  test('it should add two numbers (+)', () => {
+    expect(operate(2, 2, '+')).toBe('4');
+  });
+  test('it should subtract two numbers (-)', () => {
+    expect(operate(2, 2, '-')).toBe('0');
+  });
+  test('it should multiply two numbers (X)', () => {
+    expect(operate(2, 2, 'X')).toBe('4');
+  });
+  test('it should divide two numbers (÷)', () => {
+    expect(operate(2, 2, '÷')).toBe('1');
+  });
+  test('it should return first number if operation not described', () => {
```

Comment:
- [ ] [REQUIRED] I got an error (Uncaught Error: [big.js] Division by zero) when I divide a number by 0. Please fix it and add a test unit for this case.